documentation/API/participant-consent-datastore/openapi.json

```diff
@@ -0,0 +1,354 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "version" : "1.0",
+    "title" : "Api Documentation",
+    "contact" : { },
+    "license" : {
+      "name" : "View License",
+      "url" : "https://github.com/GoogleCloudPlatform/fda-mystudies/blob/master/LICENSE.txt"
+    }
+  },
+  "host" : "localhost:8080",
+  "basePath" : "/participant-consent-datastore",
+  "tags" : [ {
+    "name" : "Consent",
+    "description" : "Operations pertaining to consent document"
+  }, {
+    "name" : "Get Health",
+    "description" : "Provides an indication about the health of the service"
+  }, {
+    "name" : "test-pdf-upload",
+    "description" : "Test PDF Upload"
+  } ],
+  "paths" : {
+    "/consentDocument" : {
+      "get" : {
+        "tags" : [ "Consent" ],
+        "summary" : "Returns a response related to consent document",
+        "operationId" : "getStudyConsentPdfUsingGET",
+        "produces" : [ "application/json" ],
+        "parameters" : [ {
+          "in" : "body",
+          "name" : "consentVersion",
+          "description" : "consentVersion",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        }, {
+          "in" : "body",
+          "name" : "studyId",
+          "description" : "studyId",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        }, {
+          "name" : "userId",
+          "in" : "header",
+          "description" : "userId",
+          "required" : true,
+          "type" : "string"
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "schema" : {
+              "type" : "object"
+            }
+          },
+          "400" : {
+            "description" : "Invalid entries found in the submitted form. Please try again.",
+            "schema" : {
+              "$ref" : "#/definitions/ValidationErrorResponse"
+            }
+          },
+          "401" : {
+            "description" : "Unauthorized or Invalid token",
+            "schema" : {
+              "$ref" : "#/definitions/BaseResponse"
+            }
+          },
+          "500" : {
+            "description" : "Sorry, an error has occurred and your request could not be processed. Please try again later.",
+            "schema" : {
+              "$ref" : "#/definitions/BaseResponse"
+            }
+          }
+        },
+        "deprecated" : false
+      }
+    },
+    "/healthCheck" : {
+      "get" : {
+        "tags" : [ "Get Health" ],
+        "summary" : "Provides an indication about the health of the service",
+        "description" : "Default response codes 400 and 401 are not applicable for this operation",
```

Comment:
if they are not applicable to this operation, do they need to be below?

Reply:
Common response messages configured [here](https://github.com/GoogleCloudPlatform/fda-mystudies/blob/e59511e9085d986d36273eac39e762d706e623b1/common-modules/common-service/src/main/java/com/google/cloud/healthcare/fdamystudies/config/SwaggerConfig.java#L61); to avoid code duplication. 

Reference: [Global Response Messages](https://www.baeldung.com/swagger-2-documentation-for-spring-rest-api#3-custom-methods-response-messages)

Reply:
ok